### PR TITLE
Add "Report defect" Feature

### LINKED ISSUE #457: Add "Report defect" Feature
On DT1 (https://herrenberg.stadtnavi.de/) if you click anywhere on the map that is not a POI it opens up a popup that shows three actionable buttons.
image

This is not available on next branch right now
image

Please add the "Report defect" button to the pop ups. 

This PR #338 that @leonardehrenfried shared has all the necessary information to implement this.

### PR DESCRIPTION
**Pull request by jpidelatorre**
## Proposed Changes

  - Add Report defect feature to the popup that appears when clicking on an empty portion of the map

Closes #457

### CODE REVIEW

File: app/component/map/popups/LocationPopup.js [hunk 1]
```diff
@@ -121,6 +121,21 @@ class LocationPopup extends React.Component {
...
+
+    const { lat, lon } = this.props;
+
+    const reporterUrl =
+      (/^(localhost:8080|herrenberg.staging.stadtnavi.eu)$/.test(
+        window.location.host,
+      ) &&
+        `https://maengelmelder.service-bw.de/?lat=${lat}&lng=${lon}`) ||
+      null;
+    // In DT1 has this additional condition, DT2 doesn't use postal code.
```
Comment (leonardehrenfried) [inline app/component/map/popups/LocationPopup.js:133]: Could you recheck if the postal code is gone in DT2?

When I execute the following URL https://photon.stadtnavi.eu/pelias/v1/reverse?point.lat=48.59670578069282&point.lon=8.867340087890627&boundary.circle.radius=0.1&lang=de&size=1&layers=address&zones=1 I can see it.

**Comment by leonardehrenfried** [comment]
Did you manage to include the postal code?

**Comment by jpidelatorre** [comment]
> Did you manage to include the postal code?

I wanted to bring this up during the tech meeting. I can't find the properties that come from the server. They are not available to the `LocationPopup` component or the `TileLayerContainer`. I don't know if there's a service or something that I could use to retrieve that information.

**Comment by leonardehrenfried** [comment]
So, when you click somewhere on the map, then a reverse geocode request is sent. It looks something like this: https://photon.stadtnavi.eu/pelias/v1/reverse?point.lat=48.59670578069282&point.lon=8.867340087890627&boundary.circle.radius=0.1&lang=de&size=1&layers=address&zones=1

The API response contains the postalcode.

**Comment by leonardehrenfried** [comment]
The frontend already sends the query so please inspect the properties.

**Comment by jpidelatorre** [comment]
> The frontend already sends the query so please inspect the properties.

I did inspect the properties. The coordinates are in the `state` but the postal code is not in the component at all. 

image

**Comment by leonardehrenfried** [comment]
I copied the code from the master branch from here: https://github.com/stadtnavi/digitransit-ui/blob/master/app/component/map/popups/LocationPopup.js#L81

and here: https://github.com/stadtnavi/digitransit-ui/blob/master/app/component/map/popups/LocationPopup.js#L135

This gets us the postal code back.

Please check my latest commit for the exact code.

**Pull request merged**